CHANGELOG

```diff
@@ -152,6 +152,7 @@
 * Spreedly: Scrub sensitive transaction data [abarrak] #2781
 * Stripe: Add `exchange_rate` parameter [WilsonChiang] #2796
 * Mundipagg: New Gateway Implementation [nfarve] #2791
+* Beanstream: Integrated the Bambora Pacifc gateway into Bambora North America gateway #2813
```

Comment:
Can you make sure this changelog line is moved up? This version (1.78.0) was released a while ago.